Cargo.toml

```diff
@@ -8,6 +8,9 @@ license = "Apache-2.0 OR MIT"
 name = "libtock"
 repository = "https://www.github.com/tock/libtock-rs"
 version = "0.1.0"
+# Minimum required Rust compiler version for libtock-rs.
+# Aug 2023: Set to support `Result::is_ok_and()`.
+rust-version = "1.70"
```

Comment:
This should be set workspace-wide and inherited.

... which requires a newer nightly toolchain, because the current toolchain is too old to support workspace inheritance.

Reply:
Seems like _someone_ should really be looking into updating the nightly toolchain, eh? :laughing: 
I can attempt that tonight, if adding `sptr` as an external dependency is uncontroversial enough. All of the other proposals in #481 seem ... less than suboptimal.